src/providers/mod.rs

```diff
@@ -84,6 +86,7 @@ pub struct ProvidersConfig {
     pub coinbase_api_key: Option<String>,
     pub coinbase_app_id: Option<String>,
     pub one_inch_api_key: Option<String>,
+    pub getblock_access_tokens_json: Option<String>,
```

Comment:
IMO we don't need the `json` suffix here

```suggestion
    pub getblock_access_tokens: Option<String>,
```

Reply:
Makes sense, Thanks. I added the description for this variable that it's a JSON.